flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/WindowAggregateTest.scala

```diff
@@ -299,27 +299,6 @@ class WindowAggregateTest extends TableTestBase {
   }
 
   @Test
-  def testGroupWithFloorExpression() = {
-    val sql = "SELECT COUNT(*) FROM MyTable GROUP BY FLOOR(localTimestamp TO HOUR)"
```

Comment:
I think we need to discuss with the community how to distinguish non-windowed and windowed group aggregates. At the moment `GROUP BY FLOOR(rowtime() TO HOUR)` is translated into a tumbling event time window. By adding support for non-windows aggregates, it could also be executed as a such, i.e., with early firing and late update instead of a final result when the window is closed. The final result should be the same, but the behavior during execution would be different.

I think a good approach would be to only translate group window functions (TUMBLE, HOP, SESSION) into group windows and treat everything else as non-windowed aggregation. 
We should move this discussion to the mailing list though. I don't think we have to wait for a decision for this issue to continue.